packages/playwright-core/src/server/dom.ts

```diff
@@ -455,6 +455,7 @@ export class ElementHandle<T extends Node = Node> extends js.JSHandle<T> {
         // Do not await here, just in case the renderer is stuck (e.g. on alert)
         // and we won't be able to cleanup.
         hitTargetInterceptionHandle!.evaluate(h => h.stop()).catch(e => {});
+        hitTargetInterceptionHandle!.dispose();
```

Comment:
Does it mean we'll call a disposed handle at line 474 below?

Reply:
Line 454 is saying this code is exiting.